cmd/prometheus/main.go

```diff
@@ -274,6 +300,11 @@ func main() {
 		os.Exit(2)
 	}
 
+	if err := cfg.setFeatureListOptions(); err != nil {
+		fmt.Fprintln(os.Stderr, errors.Wrapf(err, "Error parsing feature list"))
```

Comment:
Should we include the passed value?

Reply:
The error returned by `setFeatureListOptions` will have the value that caused error. (`errors.Errorf("invalid feature %q", f)`)